koch.nim

```diff
@@ -559,7 +559,8 @@ proc runCI(cmd: string) =
 
   let batchParam = "--batch:$1" % "NIM_TESTAMENT_BATCH".getEnv("_")
   if getEnv("NIM_TEST_PACKAGES", "0") == "1":
-    execFold("Test selected Nimble packages", "nim r testament/testament $# pcat nimble-packages" % batchParam)
+    nimCompileFold("Compile testament", "testament/testament.nim", options = "-d:release")
+    execFold("Test selected Nimble packages", "testament $# pcat nimble-packages" % batchParam)
```

Comment:
This line verifies `testament` is in $Path.

Reply:
And package pass the test with `testament all` command as expected on Linux

![image](https://user-images.githubusercontent.com/43030857/139877693-43fa6c7d-3f68-4137-b552-6195f210c307.png)